packages/react/src/components/accordion/accordion-item.tsx

```diff
@@ -81,24 +81,24 @@ const ButtonStyled = styled(Button)<{ theme: ResolvedTheme }>`
     width: 100%;
 
     &[aria-expanded='true'] {
-        background: ${({ theme }) => theme.greys.white};
+        background: ${({ theme }) => theme.component['accordion-button-aria-extended-background-color']};
```

Comment:
Change to 'accordion-button-extended-background-color'

The CSS selector is using ARIA but the Token has no relation with Accessibility.